# Smithsonian Peer-led Bioinformatics series
Bioinformatics training series led by members of the Smithsonian community

##Videos of the talks
Users *connected to the SI Intranet* can view video links [here](http://darwin.si.edu/ads/Lists/Links/Peer-led-video.aspx). Others, not connected to the SI network, can contact us via Github for video access.

##Fall 2016
Talks are on Thursdays from 2-3pm in WG33, Natural History Building except where noted otherwise with ❗.

*Schedule is tentative and subject to change.*

Date | Speaker | Topics | Packages | Session Links
-----|---------|--------|----------|-------------------
Thurs, Oct 13 (**❗1pm**, WG33)| Matthew Kweskin & Kenneth (Tripp) Macdonald|General, R resources, task views, R studio, running R in command line/hydra; Introductory R, subsetting data, exporting importing data, data classes, some statistics||
Thurs, Oct 20 (2pm, WG33)| Kenneth (Tripp) Macdonald|Continuation of topics from week 1||
**❗Tues**, Oct 25 (2pm, WG33)| Dietrich Gotzek|Assess MCMC convergence|[CODA](https://cran.r-project.org/web/packages/coda/index.html), [BOA](http://www.public-health.uiowa.edu/boa/), and [RWTY](https://github.com/danlwarren/RWTY)|
Thurs, Nov 3 (2pm, WG33) | HC Lim|Manipulating phylogenetic trees, labeling, coloring|mainly [APE](https://cran.r-project.org/web/packages/ape/index.html)|
Thurs, Nov 10 (2pm, WG33) |Michael Lloyd|Loops, files manipulations and repetitive tasks, etc||
**❗Tues**, Nov 15 (2pm, WG33)|Caroline Judy & Andrew Gottscho|[Adegenet](https://github.com/thibautjombart/adegenet/wiki), population genetics||
**❗Tues**, Nov 22 (2pm, WG33)|Mike Trizna|Data wrangling||
Thurs, Dec 1 (2pm, WG33)|Carolyn Tepolt|[ggplot2](http://ggplot2.org/), [data.table](https://github.com/Rdatatable/data.table/wiki)||
Thurs, Dec 8 (2pm, WG33)|Nicole Angeli|Displaying and analyzing geographic data, pattern analysis, or regression/population modeling.|[phytools](https://github.com/liamrevell/phytools), [spatstat](http://spatstat.github.io/), [rgdal](https://cran.r-project.org/web/packages/rgdal/index.html), [shapefiles](https://cran.r-project.org/web/packages/shapefiles/index.html), [rgeos](https://cran.r-project.org/web/packages/rgeos/index.html), [raster](https://cran.r-project.org/web/packages/raster/index.html), and [lattice](https://cran.r-project.org/web/packages/lattice/index.html)|
Thurs, Dec 15 (2pm, WG33)|Steven Calahan|Manipulating geospatial data|[rgdal](https://cran.r-project.org/web/packages/rgdal/index.html), [ggplot2](http://ggplot2.org/)|

##Spring 2016
Talks are on Thursdays from 2-3pm in WG33, Natural History Building.

Date | Speaker | Topic | Links
-----|---------|------|------
3/17/2016 | Andy Gottscho | PyRAD - a python pipeline for Restriction-Associated-DNA sequencing | [Slides](Spring2016/Gottscho-PyRAD/pyRAD.pdf)<br/>Example on Hydra: /pool/genomics/gottschoa/test_pyrad_module<br/>[Sample param and job files](Spring2016/Gottscho-PyRAD)
3/24/2016 | Michael Trizna | Jupyter Notebook for workflow management |  [Slides](Spring2016/Trizna-Jupyter.pdf)<br/>[Example notebook](https://github.com/MikeTrizna/jupyter_notebook_presentation/blob/master/Jupyter%20Notebook%20Talk.ipynb)
3/31/2016 | Paul Frandsen | Data partitioning | [Slides](Spring2016/Frandsen-PartitionFinder/PF.pdf)<br/>[Tutorial](Spring2016/Frandsen-PartitionFinder/tutorial.md)
4/7/2016 | Rebecca Dikow | Genome Assembly | See [assembly workshop](https://github.com/SmithsonianWorkshops/GenomeAssembly)
4/14/2016 | Vanessa Gonzalez | Gene Prediction (Augustus) | [Slides](Spring2016/GonzalezVL_GenePredictionInAugustus.pdf)
4/21/2016 | Carolyn Teplot | Analysis of coexpression patterns in genes | [Slides](Spring2016/Tepolt-GenTools_popgen.pdf)

##Spring 2015
Date | Speaker | Topic | Links
-----|---------|-------|--------
4/20/2015| Rebecca Dikow | Whole genome alignment/phylogeny | [Slides](Spring2015/Dikow_WGalignment_4_19_15.pdf)
4/27/2015| Paul Frandsen | submitting to cluster with qsub, scripting | [Hydra training site](https://github.com/SmithsonianWorkshops/Hydra-workshop)
5/4/2015| HC Lim | genotype and snp calling | [Slides](Spring2015/Lim_SNP_calling.pdf), [GATK best practices videos (Broad)](http://www.broadinstitute.org/partnerships/education/broade/best-practices-variant-calling-gatk-1)
5/11/2015| Cheryl Lewis Ames | gene prediction and annotation using Trinotate | [Slides](Spring2015/Ames_Tinotate_Gene_Annotation_CLAMES_final_ed.pdf)
5/18/2015| Missy Hawkins | sequence capture experiment design | [Slides](Spring2015/Hawkins-SeqCap-Exp-Design.pdf)
6/1/2015| Matt Leray | metabarcoding data OTU calling, community-level analysis | [Slides](Spring2015/Leray-MetabarcodingAnalysis.pdf)
6/8/2015| Ehsan Kayal | Phylogenomic analysis | [Slides](Spring2015/Kayal_2015-6-8.pdf)
6/15/2015| Matthew Kweskin | Tricking out your Mac: Installing analysis software on your Mac | [Slides](Spring2015/Kweskin-Mac.pdf)
6/22/2015| Monica C. Munoz-Torres | Web Apollo & Gene Ontology Resources for Functional Annotations | [Web Apollo website](http://genomearchitect.org)
6/29/2015| Tammy Wilbert | RAD-seq library prep (pros/cons), some snp calling tools, population genomics analysis |  [Slides](Spring2015/Wilbert-RADseq_and_popgen.pdf)
